src/main/java/de/tum/cit/aet/artemis/iris/dto/IrisChatSessionDTO.java

```diff
@@ -2,11 +2,14 @@
 
 import java.time.ZonedDateTime;
 
+import jakarta.annotation.Nullable;
+import jakarta.validation.constraints.NotNull;
+
 import com.fasterxml.jackson.annotation.JsonInclude;
 
 import de.tum.cit.aet.artemis.iris.domain.session.IrisChatMode;
 
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public record IrisChatSessionDTO(Long id, Long entityId, ZonedDateTime creationDate, IrisChatMode chatMode) {
+public record IrisChatSessionDTO(@NotNull long id, @NotNull long entityId, @Nullable String entityName, ZonedDateTime creationDate, IrisChatMode chatMode) {
```

Comment:
Native types cannot be null

```suggestion
public record IrisChatSessionDTO(long id, long entityId, @Nullable String entityName, ZonedDateTime creationDate, IrisChatMode chatMode) {
```